src/core/hiwire.c

```diff
@@ -346,7 +346,17 @@ EM_JS_NUM(int, hiwire_get_length, (JsRef idobj), {
 EM_JS_NUM(bool, hiwire_get_bool, (JsRef idobj), {
   var val = Module.hiwire.get_value(idobj);
   // clang-format off
-  return (val && (val.length === undefined || val.length)) ? 1 : 0;
+  if(!val){
```

Comment:
That was not formatted by clang-format, was it? And CI passes, so something is broken there.

Reply:
See `// clang-format off` just above. It's the same problem as usual: we need to protect the javascript `===` and `!==` ops from clang-format so js code with comparisons doesn't get formatted consistently.